index.js

```diff
@@ -0,0 +1,15 @@
+/* eslint-disable no-unused-vars */
+const elem = document.getElementById('menuMobile');
+const toggleMobileMenu = () => {
+  // const logoName = document.getElementById('logo');
+  // const menuButton = document.getElementById('menuBars');
+  if (!elem.classList.contains('show-menu-mobile')) {
+    elem.classList.add('show-menu-mobile');
+  } else {
+    elem.classList.remove('show-menu-mobile');
+  }
+};
+const item = document.querySelectorAll('.list-menu-item');
```

Comment:
- [ ] It would be nice if we keep all our variables declaration at the top of the file for more readable and clean codes, kindly move this variable `const item = document.querySelectorAll('.list-menu-item')` to the top of your file

Reply:
Totally true! Thank you!

Reply:
Addressed 😁!